instrumentation-core/spring/autoconfigure/src/main/java/io/opentelemetry/spring/autoconfigure/trace/EnableOpenTelemetryTracing.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.opentelemetry.spring.autoconfigure.trace;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+
+/**
+ * Wraps {@literal @}Configuration and {@literal @}ComponentScan(basePackages =
+ * "io.opentelemetry.spring.instrumentation.trace")
+ *
+ * <p>Auto-configures OpenTelemetry instrumentation tools and interceptors</p>
```

Comment:
Shouldn't this be the summary fragment? Usually a Javadoc starts with one sentence that quickly describes the type.

Isn't the above paragraph detailing this?

Reply:
I moved this above the paragraph. This line should be a quick summery of this file